package.json

```diff
@@ -1,38 +0,0 @@
-{
-  "name": "sas-deterrence-site-generator",
```

Comment:
So when I said that these files should not be in your PR, it means that the changes to the files should not show up in the PR--as you can see both files are showing up here, but it shows that you deleted the files. That means if this PR were to be merged, it would delete the files for everyone. We need these files!

There is a difference between *deleting a file* (what you've done) and *undoing the changes you made*. 

If you refer back to the Git Immersion lessions here: http://gitimmersion.com/lab_14.html, you will see there's various types of undoing changes: 
- undo a change locally (before doing "git add")
- undoing a change locally (after "git add")
- undoing a commit (before pushing)
- undoing a commit (after pushing)

None of these are the same as deleting a file. Deleting a file is a change that gets added to our repo's history, meaning that if this was accepted, every one of us would get the change (deletion) in our local repository. 


Here is a stackoverflow that explains some of the ways to deal with this issue:
https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Try this:
1. ` git reset f2b780ab84 package.json`
2. `git status`
(you should see this:)
<img width="840" alt="screen shot 2018-09-14 at 15 32 35" src="https://user-images.githubusercontent.com/4978498/45577938-71fc4080-b833-11e8-9bf1-b6be5008745f.png">

3. Run `git diff`
You should see that the changes are all the deletions (lines that start with `-`)

4. You have the file you want staged and now the changes waiting to be staged. But the changes as shown in git diff, are actually deletions! So we want to remove the *changes* from git, not the file.

5. Run `git checkout package.json` (aka discard working changes)
6. Run `git status`
It should say 
```
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

	new file:   package.json
```
7. Commit! `git commit -m "add back package.json from revert"
8. `git push`

I think that will work for you! After you do these steps, check back at this PR and see if the `package.json` file still shows up (it shouldn't). If all goes well, follow the same process but with the `package.lock` file!

Reply:
Reset branch